mixins/_icons.scss

```diff
@@ -69,16 +73,19 @@
         content: "\64";
     }
     @if $symbol == 'down' {
-        content: "\76";
+        content: "\56";
```

Comment:
Ouch, that's a mistake. I've got the character down as `V` and it should be `v`. Thanks for spotting this.